readthedocs/projects/tasks.py

```diff
@@ -424,6 +427,14 @@ def build_docs_class(self, builder_class):
         builder.move()
         return success
 
+    def send_notifications(self):
+        """Send notifications on build failure
+
+        Don't send failure notices on ``stable`` version builds.
```

Comment:
Yeah, +1 from me, i don't remember if there was a good reason to not send stable notifies